sdk/communication/communication-sms/samples-dev/sendSmsWithMessagingConnect.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Configure Messaging Connect options to send a messages via directly customer partner and credentials
```

Comment:
The wording doesn't seem ideal, perhaps you meant this instead?
```suggestion
 * @summary Configures Messaging Connect options to send an SMS directly via a customer partner
```